RCS file: /pack/cvsroots/wxwidgets/wxWidgets/samples/minimal/minimal.cpp,v
retrieving revision 1.76
diff -u -r1.76 minimal.cpp
--- samples/minimal/minimal.cpp 2006/12/11 09:33:34     1.76
+++ samples/minimal/minimal.cpp 2007/05/01 12:29:42
@@ -40,6 +40,10 @@
     #include "../sample.xpm"
 #endif
 
+
+const wxBitmap sample(sample_xpm);
+
 // ----------------------------------------------------------------------------
 // private classes
 // ----------------------------------------------------------------------------

 	  	 
